src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -121,7 +121,7 @@ public boolean shouldImportRequestObjectParamType(FieldModel field) {
 
   @Override
   public String getRetrySettingsTypeName() {
-    return "\\Google\\ApiCore\\RetrySettings";
+    return "RetrySettings";
```

Comment:
Looking at the code, it seems very odd that this method even exists in the SurfaceNamer, since it is only implemented and used in the PhpSurfaceNamer, and in fact the RetrySettings class name is hardcoded elsewhere. WDYT of just removing this method from SurfaceNamer and PhpSurfaceNamer?